dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -189,7 +189,9 @@ def stage_extras
         @section = current_user.sections[0]
         @user = @section&.students&.find_by(id: params[:user_id])
       end
-      @show_stage_extras_warning = !@section&.stage_extras
+      # This airs on the side of not showing the warning by not showing it if the script we are in
+      # is not the assigned script for the section
```

Comment:
~airs~ --> errs

also, can we take out the double negative from the comment (as you've already done in the code): 
```
# .. by only showing it if the script we are in
# is the assigned script for the section
```